README.md

```diff
@@ -3,8 +3,29 @@
 This is implementation of Jaeger's [storage plugin](https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/grpc) for ClickHouse.
 See as well [jaegertracing/jaeger/issues/1438](https://github.com/jaegertracing/jaeger/issues/1438) for historical discussion regarding Clickhouse plugin.
 
-Note that this project is community maintained. If it is not up-to-date or missing any features
-please open the issue or submit a pull-request.
+## Project status
+
+Jaeger ClickHouse is a community-driven project, we would love to hear your feature requests.
+Pull requests also will be greatly appreciated.
+
+## Why use ClickHouse for Jaeger?
+
+[ClickHouse](https://github.com/clickhouse/clickhouse) is an analytical column-oriented database management system.
+It is designed to analyze streams of clicks which are kind of resemblant to spans.
+It's open-source, optimized for performance, and actively developed.
+
+## How it works?
+
+Jaeger spans are stored in 2 tables. First one contains whole span encoded either in JSON or Protobuf.
+Second stores key information about spans for searching. This table is indexed by span duration and tags.
+Also, info about operations is stored in the materialized view. There are not indexes for archived spans.
+Storing data in replicated local tables with distributed global tables is natively supported. Spans are bufferized.
+Span buffers are flushed to DB either by timer or after reaching max batch size. Timer interval and batch size can be
+set in [config file](https://github.com/EinKrebs/jaeger-clickhouse/blob/main/config.yaml).
+
+![Picture of tables](pictures/tables.png)
```

Comment:
How was the table generated? 

My concern is that it will get outdated as we update the schema. I would prefer generated image from commonly known tool that a community can use - or asccii picutre :)

Reply:
It was generated by JetBrains DataGrip

Reply:
you might want to document this in the readme